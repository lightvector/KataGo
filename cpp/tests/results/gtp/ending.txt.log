: Running with following config:
allowResignation = true
cudaUseFP16 = false
cudaUseNHWC = false
forDeterministicTesting = true
lagBuffer = 1.0
logAllGTPCommunication = true
logFile = tests/results/gtp/ending.txt.log
logSearchInfo = true
logSearchInfoForChosenMove = false
logTimeStamp = false
logToStderr = false
maxPlayouts = 10000
maxTimePondering = 60.0
maxVisits = 100
nnRandSeed = forTesting
nnRandomize = false
numSearchThreads = 1
openclUseFP16 = false
ponderingEnabled = false
resignConsecTurns = 3
resignThreshold = -0.90
rootSymmetryPruning = false
rules = tromp-taylor
searchFactorAfterOnePass = 0.50
searchFactorAfterTwoPass = 0.25
searchFactorWhenWinning = 0.40
searchFactorWhenWinningThreshold = 0.95
searchRandSeed = forTesting
trtUseFP16 = false

: GTP Engine starting...
: KataGo v1.16.4
: Using TrompTaylor rules initially, unless GTP/GUI overrides this
: Using 1 CPU thread(s) for search
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 19 * 19 exactly
: Cuda backend thread 0: Found GPU NVIDIA RTX A5000 memory 25425608704 compute capability major 8 minor 6
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 19 nnYLen 19
: Initializing board with boardXSize 19 boardYSize 19
: Loaded config configs/gtp_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: logDir = 
: Config override: logFile = tests/results/gtp/ending.txt.log
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: nnRandSeed = forTesting
: Config override: nnRandomize = false
: Config override: numSearchThreads = 1
: Config override: openclUseFP16 = false
: Config override: rootSymmetryPruning = false
: Config override: searchRandSeed = forTesting
: Config override: trtUseFP16 = false
: Model name: g170-b6c96-s175395328-d26788732
: GTP ready, beginning main protocol loop
: Controller: boardsize 7
: GPU -1 finishing, processed 0 rows 0 batches
: Cleaned up old neural net and bot
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 7 * 7 exactly
: Cuda backend thread 0: Found GPU NVIDIA RTX A5000 memory 25425608704 compute capability major 8 minor 6
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 7 nnYLen 7
: Initializing board with boardXSize 7 boardYSize 7
: = 
: Controller: kata-set-rules chinese
: Changed rules to Chinese
: = 
: Controller: set_position w d7 b e7 b f7 w b6 w d6 w e6 b f6 b b5 w c5 w d5 b e5 w b4 w c4 b e4 b b3 w c3 b d3 b e3 w f3 b g3 w a2 w b2 b c2 b d2 b f2 b b1
: = 
: Controller: komi 5
: = 
: Controller: genmove_debug b
: MoveNum: 0 HASH: 534D5F4EAEDAD476890520B7CAD26FA0
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O @ X . .
 3 . X O X X O X
 2 O O X X . X .
 1 . X . . . . .

koSIMPLEscoreAREAtaxNONEsui0whbNfpok1komi5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 54
NN batches: 54
NN avg batch size: 1
PV: D4 A3 F4 A1 C1 A5 pass pass pass
Tree:
: T  12.69c W  20.73c S  -2.72c ( +0.1 L  +0.1) N     100  --  D4 A3 F4 A1 C1 A5 pass
---Black(^)---
D4  : T  15.86c W  22.63c S  -2.94c ( +0.1 L  +0.1) LCB   -3.02c P 28.54% WF  55.0 PSV      73 N      54  --  D4 A3 F4 A1 C1 A5 pass pass
C1  : T   7.56c W  16.45c S  -2.65c ( +0.1 L  +0.1) LCB  -12.88c P 59.50% WF  38.9 PSV      40 N      40  --  C1 A3 D4 A1 F4 A5 pass pass
F4  : T   4.53c W  22.13c S  -1.65c ( +0.4 L  +0.2) LCB  -80.00c P  7.30% WF   4.9 PSV       3 N       5  --  F4 D4 C1 A3

: = D4
: Controller: genmove_debug w
: MoveNum: 1 HASH: 52844085B23915A6A066B24E800201DC
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O X1X . .
 3 @ X O X X O X
 2 O O X X . X .
 1 . X . . . . .

koSIMPLEscoreAREAtaxNONEsui0whbNfpok1komi5
Time taken: ###
Root visits: 100
New playouts: 46
NN rows: 25
NN batches: 25
NN avg batch size: 1
PV: A3 F4 A1 C1 A5 pass B3 E1 pass pass
Tree:
: T -17.25c W -20.20c S   0.28c ( +0.0 L  -0.0) N     100  --  A3 F4 A1 C1 A5 pass B3
---White(^)---
A3  : T -15.44c W -18.56c S   0.38c ( +0.1 L  +0.0) LCB  -29.94c P 58.04% WF  97.4 PSV      97 N      97  --  A3 F4 A1 C1 A5 pass B3 E1
C1  : T -90.39c W -85.99c S  -4.40c ( -1.3 L  -1.0) LCB -440.39c P 16.62% WF   0.8 PSV       1 N       1  --  C1
A5  : T -92.36c W -88.50c S  -3.85c ( -1.1 L  -0.8) LCB -442.36c P  4.87% WF   0.8 PSV       0 N       1  --  A5

: = A3
: Controller: genmove_debug b
: MoveNum: 2 HASH: BD17188D3FA6FC41A4A484243DEAE6FF
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O X1X @ .
 3 O2. O X X O X
 2 O O X X . X .
 1 . X . . . . .

koSIMPLEscoreAREAtaxNONEsui0whbNfpok1komi5
Time taken: ###
Root visits: 100
New playouts: 3
NN rows: 4
NN batches: 4
NN avg batch size: 1
PV: F4 A1 C1 A5 G6 C6 G5
Tree:
: T  17.30c W  19.46c S  -0.08c ( -0.1 L  -0.0) N     100  --  F4 A1 C1 A5 G6 C6 G5
---Black(^)---
F4  : T  21.24c W  23.07c S   0.10c ( -0.0 L  +0.1) LCB   -0.76c P 11.04% WF  31.8 PSV      84 N      31  --  F4 A1 C1 A5 G6 C6 G5
C1  : T  15.03c W  17.41c S  -0.18c ( -0.1 L  -0.0) LCB   -3.72c P 82.16% WF  66.4 PSV      67 N      67  --  C1 A1 F4 A5 G6 C6 G5
F5  : T -25.49c W -23.16c S  -2.33c ( -0.7 L  -0.7) LCB -375.49c P  1.19% WF   0.9 PSV       0 N       1  --  F5

: = F4
: Controller: genmove_debug w
: MoveNum: 3 HASH: 48E6C4158A76372D961D73101CF04419
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O X1X X3.
 3 O2. O X X . X
 2 O O X X . X .
 1 @ X . . . . .

koSIMPLEscoreAREAtaxNONEsui0whbNfpok1komi5
Time taken: ###
Root visits: 100
New playouts: 69
NN rows: 32
NN batches: 32
NN avg batch size: 1
PV: A1 C1 A5 pass B3 pass pass pass
Tree:
: T -21.53c W -23.02c S   0.09c ( +0.0 L  -0.0) N     100  --  A1 C1 A5 pass B3 pass pass
---White(^)---
A1  : T -20.31c W -21.72c S   0.18c ( +0.1 L  +0.0) LCB  -36.03c P 66.26% WF  93.7 PSV      93 N      97  --  A1 C1 A5 pass B3 pass pass pass
C1  : T -95.08c W -90.49c S  -4.59c ( -1.2 L  -1.3) LCB -445.08c P 17.01% WF   0.8 PSV       1 N       1  --  C1
A5  : T -76.05c W -94.01c S  -4.64c ( -1.2 L  -1.2) LCB -426.05c P  5.39% WF   0.9 PSV       0 N       1  --  A5

: = A1
: Controller: play b pass
: = 
: Controller: genmove_debug w
: MoveNum: 5 HASH: 806A1708DE637B1D9145BE60E7EE7578
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O X X X1.
 3 O . O X X . X
 2 O O X X . X .
 1 O2X @ . . . .

koSIMPLEscoreAREAtaxNONEsui0whbNfpok1komi5
Time taken: ###
Root visits: 50
New playouts: 50
NN rows: 48
NN batches: 48
NN avg batch size: 1
PV: C1 D1 B1 pass A5 pass pass pass
Tree:
: T 104.00c W  99.90c S   3.89c ( +3.5 L  +3.3) N      50  --  C1 D1 B1 pass A5 pass pass
---White(^)---
C1  : T 104.02c W  99.91c S   3.89c ( +3.5 L  +3.3) LCB  103.05c P 96.89% WF  49.0 PSV      49 N      49  --  C1 D1 B1 pass A5 pass pass pass
pss : T -53.21c W -43.41c S  -9.80c ( -0.4 L  -0.2) LCB -1400.00c P  0.15% WF   0.0 PSV       0 N       1  --  pass

: = C1
: Controller: showboard
: = MoveNum: 6 HASH: 9BE87029B630D0B6435D9AE046166F79
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O X X X .
 3 O . O X X . X
 2 O O X X . X .
 1 O1. O3. . . .
Next player: Black
Rules: {"friendlyPassOk":true,"hasButton":false,"ko":"SIMPLE","komi":5.0,"scoring":"AREA","suicide":false,"tax":"NONE","whiteHandicapBonus":"N"}
B stones captured: 2
W stones captured: 1
: Controller: final_status_list alive
: = D7 E7 F7 B6 D6 E6 F6 C5 D5 E5 B4 C4 D4 E4 F4 A3 C3 D3 E3 G3 A2 B2 C2 D2 F2 A1 C1
: Controller: final_status_list dead
: = B5
: Controller: final_score
: = W+4.0
: Controller: genmove_debug b
: MoveNum: 6 HASH: 9BE87029B630D0B6435D9AE046166F79
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O X X X .
 3 O . O X X . X
 2 O O X X . X .
 1 O1. O3@ . . .

koSIMPLEscoreAREAtaxNONEsui0whbNfpok1komi5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 0
NN batches: 0
NN avg batch size: -nan
PV: D1 B1 pass A5 F5 pass pass pass
Tree:
: T -107.28c W -99.93c S  -7.48c ( -3.5 L  -3.4) N     100  --  D1 B1 pass A5 F5 pass pass
---Black(^)---
D1  : T -107.35c W -99.94c S  -7.53c ( -3.5 L  -3.4) LCB -108.06c P 90.77% WF  99.0 PSV      99 N      99  --  D1 B1 pass A5 F5 pass pass pass

: = D1
: Controller: undo
: = 
: Controller: play b pass
: = 
: Controller: showboard
: = MoveNum: 7 HASH: 9BE87029B630D0B6435D9AE046166F79
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O X X X .
 3 O . O X X . X
 2 O O X X . X .
 1 O . O2. . . .
Next player: White
Rules: {"friendlyPassOk":true,"hasButton":false,"ko":"SIMPLE","komi":5.0,"scoring":"AREA","suicide":false,"tax":"NONE","whiteHandicapBonus":"N"}
B stones captured: 2
W stones captured: 1
: Controller: final_status_list alive
: = D7 E7 F7 B6 D6 E6 F6 C5 D5 E5 B4 C4 D4 E4 F4 A3 C3 D3 E3 G3 A2 B2 C2 D2 F2 A1 C1
: Controller: final_status_list dead
: = B5
: Controller: final_score
: = W+6.0
: Controller: kata-set-rules tromp-taylor
: Changed rules to TrompTaylor
: = 
: Controller: set_position w d7 b e7 b f7 w b6 w d6 w e6 b f6 b b5 w c5 w d5 b e5 w b4 w c4 b e4 b b3 w c3 b d3 b e3 w f3 b g3 w a2 w b2 b c2 b d2 b f2 b b1
: = 
: Controller: komi 5
: = 
: Controller: genmove_debug b
: MoveNum: 0 HASH: 534D5F4EAEDAD476890520B7CAD26FA0
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O @ X . .
 3 . X O X X O X
 2 O O X X . X .
 1 . X . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 57
NN batches: 57
NN avg batch size: 1
PV: D4 A3 F4 A1 C1 A5 pass pass
Tree:
: T   7.06c W  16.56c S  -2.65c ( +0.1 L  +0.1) N     100  --  D4 A3 F4 A1 C1 A5 pass
---Black(^)---
D4  : T  10.16c W  18.17c S  -2.82c ( +0.1 L  +0.1) LCB   -8.11c P 28.26% WF  55.2 PSV      74 N      54  --  D4 A3 F4 A1 C1 A5 pass pass
C1  : T   1.34c W  12.10c S  -2.66c ( +0.1 L  +0.1) LCB  -19.66c P 60.02% WF  38.7 PSV      38 N      40  --  C1 A3 D4 A1 F4 A5 pass pass
F4  : T   2.85c W  21.48c S  -1.64c ( +0.4 L  +0.2) LCB  -83.59c P  7.06% WF   4.9 PSV       5 N       5  --  F4 D4 C1 A3

: = D4
: Controller: genmove_debug w
: MoveNum: 1 HASH: 52844085B23915A6A066B24E800201DC
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O X1X . .
 3 @ X O X X O X
 2 O O X X . X .
 1 . X . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi5
Time taken: ###
Root visits: 100
New playouts: 46
NN rows: 12
NN batches: 12
NN avg batch size: 1
PV: A3 F4 A1 C1 A5 pass pass
Tree:
: T  -8.68c W -12.35c S   0.16c ( -0.0 L  -0.0) N     100  --  A3 F4 A1 C1 A5 pass pass
---White(^)---
A3  : T  -7.37c W -11.16c S   0.22c ( -0.0 L  -0.0) LCB  -18.54c P 59.37% WF  98.2 PSV      98 N      98  --  A3 F4 A1 C1 A5 pass pass
C1  : T -90.04c W -85.77c S  -4.27c ( -1.2 L  -1.0) LCB -440.04c P 16.20% WF   0.8 PSV       1 N       1  --  C1

: = A3
: Controller: genmove_debug b
: MoveNum: 2 HASH: BD17188D3FA6FC41A4A484243DEAE6FF
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O X1X @ .
 3 O2. O X X O X
 2 O O X X . X .
 1 . X . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi5
Time taken: ###
Root visits: 100
New playouts: 2
NN rows: 2
NN batches: 2
NN avg batch size: 1
PV: F4 A1 C1 A5 pass pass
Tree:
: T   7.46c W  10.65c S  -0.02c ( -0.0 L  -0.0) N     100  --  F4 A1 C1 A5 pass pass
---Black(^)---
F4  : T  11.56c W  14.23c S   0.14c ( +0.0 L  +0.1) LCB   -5.47c P 11.03% WF  37.0 PSV      93 N      36  --  F4 A1 C1 A5 pass pass
C1  : T   4.27c W   7.88c S  -0.13c ( -0.0 L  -0.0) LCB   -9.54c P 82.37% WF  61.1 PSV      62 N      62  --  C1 A1 F4 A5 pass pass
F5  : T -26.34c W -23.87c S  -2.47c ( -0.7 L  -0.7) LCB -376.34c P  1.15% WF   0.9 PSV       0 N       1  --  F5

: = F4
: Controller: genmove_debug w
: MoveNum: 3 HASH: 48E6C4158A76372D961D73101CF04419
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O X1X X3.
 3 O2. O X X . X
 2 O O X X . X .
 1 @ X . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi5
Time taken: ###
Root visits: 100
New playouts: 64
NN rows: 21
NN batches: 21
NN avg batch size: 1
PV: A1 C1 A5 pass pass
Tree:
: T  -7.28c W  -9.29c S   0.17c ( +0.0 L  -0.0) N     100  --  A1 C1 A5 pass pass
---White(^)---
A1  : T  -5.83c W  -7.74c S   0.25c ( +0.0 L  +0.0) LCB  -15.88c P 66.29% WF  97.4 PSV      97 N      97  --  A1 C1 A5 pass pass
C1  : T -94.51c W -90.15c S  -4.37c ( -1.2 L  -1.3) LCB -444.51c P 17.18% WF   0.8 PSV       1 N       1  --  C1
A5  : T -71.70c W -93.76c S  -4.45c ( -1.2 L  -1.1) LCB -421.70c P  5.37% WF   0.8 PSV       0 N       1  --  A5

: = A1
: Controller: play b pass
: = 
: Controller: genmove_debug w
: MoveNum: 5 HASH: 806A1708DE637B1D9145BE60E7EE7578
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O X X X1.
 3 O . O X X . X
 2 O O X X . X .
 1 O2X @ . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi5
Time taken: ###
Root visits: 50
New playouts: 50
NN rows: 39
NN batches: 39
NN avg batch size: 1
PV: C1 D1 B1 pass A5 pass pass
Tree:
: T 104.25c W  99.90c S   4.06c ( +3.6 L  +3.4) N      50  --  C1 D1 B1 pass A5 pass pass
---White(^)---
C1  : T 104.27c W  99.91c S   4.07c ( +3.6 L  +3.4) LCB  103.33c P 96.95% WF  49.0 PSV      49 N      49  --  C1 D1 B1 pass A5 pass pass
pss : T -51.42c W -41.64c S  -9.78c ( -0.3 L  -0.2) LCB -1400.00c P  0.16% WF   0.0 PSV       0 N       1  --  pass

: = C1
: Controller: undo
: = 
: Controller: play w pass
: = 
: Controller: showboard
: = MoveNum: 6 HASH: 806A1708DE637B1D9145BE60E7EE7578
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O X X X .
 3 O . O X X . X
 2 O O X X . X .
 1 O1X . . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":5.0,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 1
W stones captured: 1
: Controller: final_status_list alive
: = D7 E7 F7 B6 D6 E6 F6 B5 C5 D5 E5 B4 C4 D4 E4 F4 A3 C3 D3 E3 G3 A2 B2 C2 D2 F2 A1 B1
: Controller: final_status_list dead
: = 
: Controller: final_score
: = B+9.0
: Controller: genmove_debug b
: MoveNum: 6 HASH: 806A1708DE637B1D9145BE60E7EE7578
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O X X X .
 3 O . O X X . X
 2 O O X X . X .
 1 O1X @ . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi5
Time taken: ###
Root visits: 50
New playouts: 50
NN rows: 2
NN batches: 2
NN avg batch size: 1
PV: C1 A5 pass pass
Tree:
: T   9.32c W  11.97c S  -1.10c ( +0.0 L  +0.0) N      50  --  C1 A5 pass pass
---Black(^)---
C1  : T   8.61c W  11.32c S  -1.14c ( +0.0 L  +0.0) LCB   -6.94c P 97.04% WF  49.0 PSV      49 N      49  --  C1 A5 pass pass
pss : T -112.20c W -99.37c S -12.83c ( -3.5 L  -3.0) LCB -1400.00c P  0.97% WF   0.0 PSV       0 N       1  --  pass

: = C1
: Controller: undo
: = 
: Controller: play b pass
: = 
: Controller: showboard
: = MoveNum: 7 HASH: 806A1708DE637B1D9145BE60E7EE7578
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O X X X .
 3 O . O X X . X
 2 O O X X . X .
 1 O X . . . . .
Next player: White
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":5.0,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 1
W stones captured: 1
: Controller: final_status_list alive
: = D7 E7 F7 B6 D6 E6 F6 B5 C5 D5 E5 B4 C4 D4 E4 F4 A3 C3 D3 E3 G3 A2 B2 C2 D2 F2 A1 B1
: Controller: final_status_list dead
: = 
: Controller: final_score
: = B+9.0
: GPU -1 finishing, processed 789 rows 789 batches
: All cleaned up, quitting
