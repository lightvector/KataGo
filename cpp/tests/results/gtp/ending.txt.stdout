= 

= 

= 

= 

= C1

= A3

= A1

= D4

= 

= pass

= MoveNum: 6 HASH: FA804294DA0212079F6C04393DD0BE70
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O O1X . .
 3 O . O X X O X
 2 O O X X . X .
 1 X X X . . . .
Next player: Black
Rules: {"friendlyPassOk":true,"hasButton":false,"ko":"SIMPLE","komi":5.0,"scoring":"AREA","suicide":false,"tax":"NONE","whiteHandicapBonus":"N"}
B stones captured: 1
W stones captured: 0

= D7 E7 F7 B6 D6 E6 F6 C5 D5 E5 B4 C4 D4 E4 A3 C3 D3 E3 G3 A2 B2 C2 D2 F2 A1 B1 C1

= B5 F3

= 0

= pass

= 

= 

= MoveNum: 7 HASH: FA804294DA0212079F6C04393DD0BE70
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O O X . .
 3 O . O X X O X
 2 O O X X . X .
 1 X X X . . . .
Next player: White
Rules: {"friendlyPassOk":true,"hasButton":false,"ko":"SIMPLE","komi":5.0,"scoring":"AREA","suicide":false,"tax":"NONE","whiteHandicapBonus":"N"}
B stones captured: 1
W stones captured: 0

= D7 E7 F7 B6 D6 E6 F6 C5 D5 E5 B4 C4 D4 E4 A3 C3 D3 E3 G3 A2 B2 C2 D2 F2 A1 B1 C1

= B5 F3

= 0

= 

= 

= 

= C1

= A3

= A1

= D4

= 

= A5

= 

= 

= MoveNum: 6 HASH: FA804294DA0212079F6C04393DD0BE70
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O O1X . .
 3 O . O X X O X
 2 O O X X . X .
 1 X X X . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":5.0,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 1
W stones captured: 0

= D7 E7 F7 B6 D6 E6 F6 B5 C5 D5 E5 B4 C4 D4 E4 A3 C3 D3 E3 F3 G3 A2 B2 C2 D2 F2 A1 B1 C1

= 

= B+1.0

= F4

= 

= 

= MoveNum: 7 HASH: FA804294DA0212079F6C04393DD0BE70
   A B C D E F G
 7 . . . O X X .
 6 . O . O O X .
 5 . X O O X . .
 4 . O O O X . .
 3 O . O X X O X
 2 O O X X . X .
 1 X X X . . . .
Next player: White
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":5.0,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 1
W stones captured: 0

= D7 E7 F7 B6 D6 E6 F6 B5 C5 D5 E5 B4 C4 D4 E4 A3 C3 D3 E3 F3 G3 A2 B2 C2 D2 F2 A1 B1 C1

= 

= B+1.0

